specs/AppNotifications/AppNotifications-spec.md

```diff
@@ -139,6 +139,29 @@ int main()
 }
 ```
 
+## Registering for App Notifications using assets
+
+For Unpackaged applications, the developer calls into Register API with DisplayName and Icon.
+WinAppSDK will register and display them when an App Notification is received. The developer
```

Comment:
nit: Windows App SDK

Also, some rewording will help to express the idea here (which I think is it registering the app and then display a notification?).